code/modules/power/supermatter/supermatter.dm

```diff
@@ -90,6 +90,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 
 	critical_machine = TRUE
 
+	var/list/gas_trans = list("PL" = PLASMA_TRANSMIT_MODIFIER, "O2" = OXYGEN_TRANSMIT_MODIFIER, "TRIT" = TRITIUM_TRANSMIT_MODIFIER, "PLX" = PLUOXIUM_TRANSMIT_MODIFIER, "BZ" = BZ_TRANSMIT_MODIFIER)
```

Comment:
You should probably just index the whole thing by gas types but it's not really important.

Reply:
Yeah, will do that at some point, but I'd rather do that for all the gas usages in the function. So work for another day.